scripts/support_tunnel_post_down.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# This script runs after a sdownport tunnel comes down. Right now, 
```

Comment:
after a support tunnel comes down?

Reply:
lmfao, this was when I copied the `post-up` script and just ran `s/up/down/` :woozy_face: